src/http.rs

```diff
@@ -701,6 +706,149 @@ fn status_active(
     .unwrap()
 }
```

Comment:
From the documentation, it sounds like this will only fail if you fed the builder bad data. Since in this case, the only bad data would be the Content-Length header, blowing up seems appropriate?

Reply:
I don't know, it leaves the client without a response at all, and then it has to second guess what happened. I think it's preferable to say 500, "I blew up, but I managed to report this back to you". Even if it's the last thing you're saying. Client can then confidently state 'Server blew itself up' to the user. I think http servers are different in that respect than cli tools or daemons that are ran by end-users, they can inspect logs and cli panic output and what not. HTTP end clients have to make do with the HTTP response only, everything else is second guessing. (Sorry for the long answer)

Reply:
All valid points except, how do I make that 500? It’ll be a `ResponseBuilder` on which I will have to call `body` all over again …

Reply:
Hm, okay, that’s not actually true, `Response::from_parts` would be an option that totally avoids the builder.

Reply:
I created #450 and will leave it as is for now.